src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html

```diff
@@ -8,12 +8,12 @@
     (onFilterTypeAhead)="filterQueries($event)">
   </pfng-toolbar>
 </div>
-<div class="active-filters-section">  
+<div class="f8-filters">  
   <span class="dib margin-right-10"
     *ngIf="activeFilterFromSidePane !== ''">
     <strong>{{totalCount | async}}</strong> in {{activeFilterFromSidePanel}}
   </span>
-  <div class="active-filters-div"
```

Comment:
@SMahil Work Item datatable list: should filter work item by type is waiting for class ".active-filters-div"
Please update this:
https://github.com/fabric8-ui/fabric8-planner/blob/master/src/tests/ui/planner/toolbarHeader.ts#L26